models/model_train/transfer-learn.ipynb

```diff
@@ -0,0 +1,614 @@
+    "\n",
+    "- The embedding layer takes the input image, video, or text, and converts it into a vector of numbers, where each number represents the similarity of the input to a particular word or concept. The classifier layer then takes the embedding vector as input and predicts the class of the input.\n",
+    "\n",
+    "- Essentially, transfer learning leverages the knowledge gained from a pre-trained model to facilitate the learning process of a new model for a related problem. The pre-trained embedding model serves as a feature extractor that has already learned useful features from a large dataset. This knowledge is transferred to the classifier layers, enabling effective classification with minimal training data.\n",
+    "\n",
+    "- With transfer-learning models, you can transfer the learnings from existing models to your own custom models. This means that you can come up with your own set of custom concepts and quickly train a new model with relatively few training samples.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c45434c0",
+   "metadata": {},
+   "source": [
+    "Note: The base model version has to be one of the embed models in the app workflow to use for embeddings. The user can specify the specific model in training params incase the default workflow of your app has multiple embedding models present in it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e6e1f5ec",
+   "metadata": {},
+   "source": [
+    "### Creating an App"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "ec535aed",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from clarifai.client.user import User\n",
+    "#replace your \"user_id\"\n",
+    "client = User(user_id=\"user_id\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a9d27285",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app = client.create_app(app_id=\"transfer_learn\", base_workflow=\"Universal\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d93d5ddc",
+   "metadata": {},
+   "source": [
+    "### Uploading Image Dataset with concepts"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "97c6cdfd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "module_path = os.path.join(os.getcwd().split('/models/model_train')[0],'datasets/upload/image_classification/food-101')\n",
+    "module_path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "d8b58bc6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "# adding dataset.py folder in food-101 folder to the system path for dataloader object\n",
+    "sys.path.insert(0, module_path)"
```

Comment:
we could use `load_module_dataloader` functionality
https://github.com/Clarifai/clarifai-python/blob/4c75dd673615990ce4af5b05a13d0552e1b2402a/clarifai/datasets/upload/utils.py#L14

Just the path of module gives dataloader

Reply:
Added those..